src/colorizer/Dataset.ts

```diff
@@ -591,7 +591,10 @@ export default class Dataset {
       }, [] as number[]);
     }
 
-    const track = new Track(trackId, times, ids, centroids, bounds);
+    let track = null;
+    if (ids.length > 0) {
+      track = new Track(trackId, times, ids, centroids, bounds);
+    }
```

Comment:
Explicitly return `null` if the track is empty. Before, if a track didn't exist in the Dataset, a Track with no IDs would be returned. This required the client to know that this behavior was possible and handle it as an error case.